### PR TITLE
Search People Feature

### LINKED ISSUE #7634: Search feature in People section
There should be a search feature in https://publiclab.org/people as it become difficult to find a user manually.
Screenshot from 2020-03-12 23-15-13
Thanks

### PR DESCRIPTION
**Pull request by Cadreia**
Fixes #7634
search_pl

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

File: app/views/tag/show/_tab_content.html.erb [hunk 1]
```diff
@@ -66,6 +66,21 @@
         <%= render partial: "tag/contributors" %>
       </div>
   <% end %>
-  <br />
+  <br>
 
 </div>
+
```
Comment (sagarpreet-chadha) [inline app/views/tag/show/_tab_content.html.erb:72]: Why is this change required? Was this not happening via CSS?

Comment (Cadreia) [reply app/views/tag/show/_tab_content.html.erb:72]: It wasn't working via CSS only, hence the bug. I've not been able to figure out why yet, but it works fine now with the jquery code I included.

Comment (Cadreia) [reply app/views/tag/show/_tab_content.html.erb:72]: This file was modified to solve a different issue, not this one. But the code wasn't working with just the css. so I had to include the jquery code.

File: app/models/user.rb [hunk 2]
```diff
@@ -410,6 +410,14 @@ def search_by_username(query)
       User.where('MATCH(username) AGAINST(? IN BOOLEAN MODE)', "#{query}")
     end
 
+    def search_people(query)
+      if query
+        User.where('username LIKE ?', "%#{query}%")
```
Comment (jywarren) [inline app/models/user.rb:415]: I'm just a little worried about the performance of this query, as our user table is really huge due to spam users. I'm going to suggest some refinements as a suggestion, and then I think we can merge this! Starting with, only using wildcards on the end of the search query, to make a more optimized db search query. Then, filtering by user status! Thanks!

```suggestion
        User.where('username LIKE ?', "#{query}%")
          .where(status: 1)
```

Comment (Cadreia) [reply app/models/user.rb]: Okay, sounds good to me! Thanks for the review

**Review by sagarpreet-chadha — APPROVED** [review]
Sorry for late review, this looks great 💯

**Comment by cesswairimu** [comment]
Hi @Cadreia, kindly rebase this to retrigger test-runner. Thanks
